.github/dependabot.yml

```diff
@@ -9,3 +9,7 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: weekly
```

Comment:
Will this work with hash-pinned actions?

I suppose it can be useful for github official actions pinned using semver 🤷 👍